napari/utils/transforms/transforms.py

```diff
@@ -491,6 +500,38 @@ def ndim(self) -> int:
         """Dimensionality of the transform."""
         return self._linear_matrix.shape[0]
 
+    @property
+    def axis_labels(self) -> tuple[str, ...]:
+        """tuple of axis labels for the layer."""
+        return self._axis_labels
+
+    @axis_labels.setter
+    def axis_labels(self, axis_labels: Optional[Sequence[str]]) -> None:
+        if axis_labels is None:
+            axis_labels = tuple(f'axis {i}' for i in range(-self.ndim, 0))
```

Comment:
I'm not sure here, just a proposal for discussion:
1. `'axis -i'` is one option (current option)
2. *Just* `'-i'`
3. directly integers instead of strings, so they can be used for indexing.

I'm not sure at all of the merits of each option — I'm proposing only because in my brain I probably would have used option (3). I suspect that it does not much matter because we will never be able to actually to use them directly — we'll need to write code that does different things depending on the input. However, for example, it might be interesting to use just the integers, because then you know whether you are using the default or something set by the metadata. The same can be said of option 2, if we for example forbid strings starting with `-` from users.

Thoughts?

Reply:
I do not want ints. I prefer to keep it sting only. Code will be simpler and easier to avoid errors.

Reply:
Ok. How about option 2 then?

Reply:
I prefer `axis -1` as it is less possible to make confusion when printing. But do not have a strong opinion.

Reply:
I just think that at some point these are going to end up on the dims controls on the viewer, and "axis -1" takes up so much space/is ugly. I agree that it's less ambiguous, though.

Since currently we are not using this information anywhere, I am going to merge this and make an issue about it. (After merging in main to get CI passing again.)

Reply:
I think that in some moment, we will land in ordering `t`, `z`, `x`, `y` based on units metadata with addition of `p`, `m`, `s`  (or other letters used in tiff metadata)